.nvmrc

```diff
@@ -0,0 +1 @@
+10.15
```

Comment:
You should not need that.
Please make sure you have a similar setup to this project
https://github.com/ardeois/graphql-codegen-typescript-mock-data